class/subscription.php

```diff
@@ -1195,4 +1197,23 @@ public function insert_free_pack_subscribers( $pack_id, $userdata ) {
             $wpdb->insert( $wpdb->prefix . 'wpuf_subscribers', $table_data );
         }
     }
+
+    /**
+     * Reset trials data if used once
+     *
+     * @since WPUF_PRO
+     *
+     * @param $sub_meta
+     *
+     * @return mixed
+     */
+    public function reset_trial_if_used_once( $sub_meta ) {
```

Comment:
@zamandevs you can use the function name as `reset_trial` instead of `reset_trial_if_used_once`

Reply:
The function name suggested by @hasinur1997 seems more logical.